docs/content/ingestion/data-formats.md

```diff
@@ -105,10 +110,17 @@ The `columns` field must match the columns of your input data in the same order.
   }
 ```
 
-The `columns` field must match the columns of your input data in the same order. 
+#### TSV (Delimited) Index Tasks
+
+If your file does not have a header as the first line of the file, you must set the `columns` field and ensure that the order of the fields matches the columns of your input data in the same order.
+If your file does have a header, you can set a field called `hasHeaderRow` to true, and do not include the `columns` key.
 
 Be sure to change the `delimiter` to the appropriate delimiter for your data. Like CSV, you must specify the columns and which subset of the columns you want indexed.
```

Comment:
This `delimiter` note applies to all kinds of tasks, but being under this header suggests it only applies to "index" tasks.